app/views/tube_racks/summaries/_default.html.erb

```diff
@@ -7,12 +7,15 @@
     <li class="nav-item">
       <a class="nav-link active" href="#summary_tab" data-toggle="tab" role="tab" data-plate-view="summary-view">Summary</a>
     </li>
-    <% data_attributes = {
-        sequencescape_api: Limber::Application.config.api.v2.connection_options.js_url,
-        sequencescape_api_key: Limber::Application.config.api.v2.connection_options.authorisation,
-        asset_id: presenter.id
-    } %>
     <li class="nav-item">
+      <a class="nav-link" href="#relatives_tab" data-toggle="tab" role="tab" data-plate-view="pools-view">Relatives</a>
+    </li>
+    <li class="nav-item">
+      <% data_attributes = {
+          sequencescape_api: Limber::Application.config.api.v2.connection_options.js_url,
+          sequencescape_api_key: Limber::Application.config.api.v2.connection_options.authorisation,
```

Comment:
It is not specific to this PR but we are including the key in page sources. Maybe we can discuss if there is any alternative to this.